e2e_tests/src/e2e_conduit/runner.py

```diff
@@ -0,0 +1,103 @@
+import atexit
+import os
+import logging
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+
+import yaml
+
+from e2e_common.util import find_binary
+from e2e_conduit.subslurp import subslurp
+
+logger = logging.getLogger(__name__)
+
+
+class ConduitE2ETestRunner:
+    def __init__(self, conduit_bin, keep_temps=False):
+        self.conduit_bin = find_binary(conduit_bin, binary_name="conduit")
+        self.keep_temps = keep_temps
+
+    def setup_scenario(self, scenario):
+
+        # Setup conduit_dir for conduit data dir
+        scenario.conduit_dir = tempfile.mkdtemp()
+        if not self.keep_temps:
+            atexit.register(shutil.rmtree, scenario.conduit_dir, onerror=logger.error)
+        else:
+            logger.info("leaving temp dir %r", scenario.conduit_dir)
```

Comment:
I'm not seeing the reason to print`scenario.coundit_dir`'s `repr()` with `%r` rather than the more typical `%s` which uses `str()`.